test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +44,29 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNullTest() throws Exception {
+        //empty
+        assertNoNull();
+
+        //no null
+        assertNoNull(123, "a String", "");
+
+        //some null
+        assertWithNull(null, 1234, "a String", null);
+        assertWithNull(null, null, null, 123);
+
+        //all null
+        assertWithNull(null);
+        assertWithNull(null, null, null, null);
+    }
+
+    private void assertNoNull(Object... objects) {
+        assertTrue(Utils.isAnyNull(objects));
```

Comment:
You might have confused yourself. What this method seems to be doing is asserting that there is indeed `null`, contrary to the method signature.